lib/processCss.js

```diff
@@ -158,18 +161,18 @@ module.exports = function processCss(inputSource, inputMap, options, callback) {
 	var pipeline = postcss([
 		localByDefault({
 			mode: options.mode,
-			rewriteUrl: function(global, url) {
+			rewriteUrl: function(global, origUrl) {
 				if(parserOptions.url){
-					url = url.trim(" ");
+					var url = origUrl.trim(" ");
 
-					if(!loaderUtils.isUrlRequest(url, root)) {
-						return url;
+					if(!origUrl.replace(/\s/g, '').length || !loaderUtils.isUrlRequest(url, root)) {
+						return origUrl;
```

Comment:
@michael-ciniawsky i think we don't change original value of `url` (example `background: url('     ')` many spaces :smile: ) should be `background: url('     ')` after processing, if we return trimmed, we get `background: url('')` output (as now). I would not call it a bug, but this is a non-standard behavior that in theory we should not do.